src/sentry/services/hybrid_cloud/auth/impl.py

```diff
@@ -184,7 +184,9 @@ def authenticate(self, *, request: AuthenticationRequest) -> MiddlewareAuthentic
             auth = AuthenticatedToken.from_token(fake_request.auth)
 
         result = MiddlewareAuthenticationResponse(
-            auth=auth, user_from_signed_request=fake_request.user_from_signed_request
+            auth=auth,
+            user_from_signed_request=fake_request.user_from_signed_request,
+            accessed=fake_request.session._accessed,
```

Comment:
Should there be a non `_` prefixed property to get the list of accessed keys?

Reply:
Not sure what you mean?  In the case of the response, it is non prefixed so that it is public and available.  It is only prefixed in `FakeRequestDict` which is an implementation detail.